### PR TITLE
fixed spacing in Spam dashboard widget where number of trapped comments was on top of text

### PR DESCRIPTION
**Pull request by ops-andy**
## Overview
fixed spacing in Spam dashboard widget where number of trapped comments was on top of text

## Nature of This Change

- [ ] 🐛 Fixes a bug
- [ ] 🚀 Implements a new feature
- [x] 🛁 Refactors existing code
- [ ] 💅 Fixes coding style
- [ ] ✅ Adds tests
- [ ] 👽 Adds new dependency
- [ ] 🔥 Removes unused files / code
- [ ] 🔒 Improves security

### CODE REVIEW

File: system/ee/ExpressionEngine/View/_shared/dashboard/dashboard.php [hunk 1]
```diff
@@ -181,7 +181,7 @@
 		<ul class="simple-list">
 		<?php foreach ($trapped_spam as $trapped): ?>
 			<li><a href="<?=ee('CP/URL')->make('addons/settings/spam', array('content_type' => $trapped->content_type))?>">
-				<b><?=$trapped->total_trapped?></b> <?=lang($trapped->content_type)?> <?=lang('spam')?>
+			<b class="ml-s"><?=$trapped->total_trapped?></b>&nbsp; <?=lang($trapped->content_type)?><?php if ($trapped->total_trapped > 1):?>s<?php endif ?> marked as <?=lang('spam')?>
```
Comment (intoeetive) [inline system/ee/ExpressionEngine/View/_shared/dashboard/dashboard.php]: "marked as" needs to be moved to lang file

**Review by intoeetive — CHANGES_REQUESTED** [review]

**Comment by obfuscode** [comment]
@ops-andy Can you extract the "marked as" text to the language file. It also looks like one of the tests is failing but that may be a faulty run. We can take a look again after the language file update if it's still failing.

**Comment by ops-andy** [comment]
@obfuscode updated to fix both plural issue and moving 'marked_as'

**Comment by ops-andy** [comment]
@obfuscode once again I'm getting timeout with the removing variable step in the tests. Since my code doesn't touch that, I think this should be good to go.

**Review by obfuscode — APPROVED** [review]

**Pull request merged**